### PR TITLE
Initial Reconfiguration and Structuring

### PR DESCRIPTION
**Pull request by DaintyJet**
Reorganized files to make the root directory less cluttered

Added an OS-Service Directory:
* The LINUX directory is still located in the root since there are branches with changes made to it.
* Moved Windows into this; once outstanding branches are merged, we will move the LINUX directory into the OS-Service directory

Modified Ansible Directory:
* Added Linux Directory
* Added Windows Directory
* Added Directory for all others
> Although Ansible playbooks can account for operating systems themselves, keeping it simple is likely better at this point.

Added Task-Management-Workflow Directory:
* Moved Taiga Directory into this along with its related Images
* Moved GitHub Style Guide into this along with its Related Images

Added Working-ccdc2024:
To prevent issues with the [submodules](https://github.blog/2016-02-01-working-with-submodules/) this is just a straight copy-paste of the ccdc2024 repository. I can revert this to be a submodule.

Added Injects-Incidents-Infrastructure:
* Moved Proxmox documentation into this Directory
* Service-First-15 Directory is for the documentation relating to the first 15 mins of a service
* Service-Possible-Incidents is for possible incidents (attack vectors) relating to a service
* Service-Possible-Injects is for possible Injects relating to your service.
* Templates still contain our own templates

### CODE REVIEW

**Comment by P-rple** [comment]
Regarding this:
   - The LINUX directory is still located in the root since there are branches with changes made to it.
    - Moved Windows into this; once outstanding branches are merged, we will move the LINUX directory into the OS-Service directory
> Lets make this a task with a deadline, so we can get the repository as update and finalize as possible, I propose Tuesday, since


Under Scripts > "OS Name" > Maybe we should have the directory for scripts based on services
In addition make a folder called "Hardening Service"  and place hardening scripts in there, or something similar to that idea
> Task: Each person can create their own service folder name than make a pull request to get it updated.

Alternate option: Make a new directory of scripts Under the (i.e) OS Name > Service > Scripts.
Reasoning: Is there ever a time where scripts will not be run relative to the OS, or service, other than Ansible. Which I agree, Ansible should have its own folder, for simplicity.

File: Injects-Incidents-Infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Injects-Incidents-Infrastructure <!-- omit-from-toc -->
+This directory contains materials relating to possible injects, incidents and the infrastructure (it's setup) that we are using. Of note should be the section [Writing Guide](#writing-guide) since this is the main reason this README was made.
```
Comment (P-rple) [inline Injects-Incidents-Infrastructure/README.md:2]: ...are using. of note...  Typo, I would just change it but I don't really know what it means.

**Review by P-rple — APPROVED** [review]
Just one Typo. I don't care whether its fixed or not, the point and purpose of the document is still properly conveyed.

**Comment by DaintyJet** [comment]
> Regarding this:
> 
> * The LINUX directory is still located in the root since there are branches with changes made to it.
>   - Moved Windows into this; once outstanding branches are merged, we will move the LINUX directory into the OS-Service directory
> 
> > Lets make this a task with a deadline, so we can get the repository as update and finalize as possible, I propose Tuesday, since
> 
> Under Scripts > "OS Name" > Maybe we should have the directory for scripts based on services In addition make a folder called "Hardening Service" and place hardening scripts in there, or something similar to that idea
> 
> > Task: Each person can create their own service folder name than make a pull request to get it updated.
> 
> Alternate option: Make a new directory of scripts Under the (i.e) OS Name > Service > Scripts. Reasoning: Is there ever a time where scripts will not be run relative to the OS, or service, other than Ansible. Which I agree, Ansible should have its own folder, for simplicity.

I am fine with a deadline, up to you all.

For the Scripts, I did (I think) add a directory Scripts after the pull request had been made start with commit [c229698](https://github.com/UML-Cyber-Security/CCDC_Practice_Infrastructure/pull/37/commits/c2296986c3d8f7de17407647662cd51bbf3c80aa)

**Comment by ChrisM09** [comment]
Do you want me to review and merge this  _now_ or are we making changes and then after the deadline you talked about I review? @P-rple

**Comment by DaintyJet** [comment]
If you think there is a typo that needs to be fixed, or any of the examples or guidelines need to be updated they can be made here before the merge.

Otherwise it can be merged.

**Comment by P-rple** [comment]
> Do you want me to review and merge this _now_ or are we making changes and then after the deadline you talked about I review? @P-rple

We can merge it, those were just suggestions, if Matt wanted them. I have no issues with this.

**Review by ChrisM09 — APPROVED** [review]
Definitely read through 1,925 files. LGTM

**Pull request merged**